bridge/src/client/data_store/local_file.rs

```
@@ -0,0 +1,156 @@
+use crate::{
+    error::err_to_string,
+    utils::{compress, decompress, DEFAULT_COMPRESSION_LEVEL},
+};
+
+use super::base::DataStoreDriver;
+use async_trait::async_trait;
+use dotenv;
+
+// To use this data store, create a .env file in the base directory with the following values:
+// export BRIDGE_FILE_ROOT_PATH="..."
+pub struct LocalFile {
+    base_path: std::path::PathBuf,
+}
+
+impl LocalFile {
+    pub fn new() -> Option<Self> {
+        dotenv::dotenv().ok();
+        let root_path = dotenv::var("BRIDGE_FILE_ROOT_PATH");
+
+        if root_path.is_err() {
+            return None;
```

We shouldn't silently fail on an error a user can fix.

---

It's the design of data_store, if all data store drivers are None, data store will throw an Error. We also can use this feature to choose which driver to use by enable/disable vars in .env